doc-templates/sandbox/VehicleRentalServiceDirectory.md

```diff
@@ -0,0 +1,38 @@
+# Vehicle Rental Service Directory API support.
+
+This adds support for the GBFS service directory endpoint component located
+at https://github.com/entur/lamassu. OTP use the service directory to lookup and connect to all GBFS
```

Comment:
```suggestion
at https://github.com/entur/lamassu. OTP uses the service directory to lookup and connect to all GBFS
```

Reply:
I think this is correct (not *lookup and connect***s**)?

> OTP uses the service directory to lookup and connect to all GBFS endpoints registered in the directory.

Reply:
Your right. The second suggestion is wrong.